configuration/docker-compose.yml

```diff
@@ -15,13 +15,15 @@ services:
     volumes:
       - ./mysql/localhost.sql:/docker-entrypoint-initdb.d/localhost.sql
       - ./mysql/port_drayage.sql:/docker-entrypoint-initdb.d/port_drayage.sql
-      # The following 3 volumes are used for CDA1Tenth. By default the standard "port_drayage.sql" file
+      # The following 5 volumes are used for CDA1Tenth. By default the standard "port_drayage.sql" file
       # will be loaded. Comment out the "port_drayage.sql" file above and uncomment the following 
       # three files to enable the CDA1Tenth sql files to be loaded. The container volumes MUST be reset
       # to allow for this change, which can be done with "docker compose down -v"
```

Comment:
1) "three files" -> "five files"
2) Did you end up having to do the -v reset to load the new sql files for testing? if not we can remove this, it may no longer be relevant instruction